src/style.css

```
@@ -1 +1,18 @@
-/* styles go here */
+.personal-information,
+.registration,
+.facts,
+.info {
+  display: block;
+}
+
+label {
+  margin-bottom: 10px;
+}
+
+fieldset label:last-child {
+  margin-bottom: 0;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
```

1. Use class selectors instead of tag selectors to reach better performance

```suggestion
.fieldset {
  margin-bottom: 20px;
}

.label {
  margin-bottom: 10px;
}

.fieldset .label:last-child {
  margin-bottom: 0;
}
```